test/unit/search/dream_index/interleaved_bloom_filter_test.cpp

```diff
@@ -157,6 +157,11 @@ TYPED_TEST(interleaved_bloom_filter_test, counting)
     }
     std::vector<size_t> expected(128, 128);
     EXPECT_EQ(counting, expected);
+
+    // Counting vectors can be added together
+    std::vector<size_t> expected2(128, 256);
```

Comment:
Why is it not (256,256)?

Reply:
Because the size of the vector does not change, i.e. it's

```
  [128, 128, 128, ..., 128] // size 128
+ [128, 128, 128, ..., 128] // size 128
---------------------------------
  [256, 256, 256, ..., 256] // size 128
```